shared/metrics/src/main/java/io/pravega/shared/MetricsNames.java

```diff
@@ -95,14 +95,14 @@
     public static final String CACHE_GET_LATENCY = "segmentstore.cache.get_latency_ms";         // Histogram
     public static final String CACHE_READ_BYTES = "segmentstore.cache.read_bytes";              // Counter
     public static final String CACHE_TOTAL_SIZE_BYTES = "segmentstore.cache.size_bytes";        // Gauge
-    public static final String CACHE_GENERATION_SPREAD = "segmentstore.cache.gen";              // Histogram
+    public static final String CACHE_GENERATION_SPREAD = "cache.gen";                           // Histogram
 
     // DurableDataLog (Tier1) stats
-    public static final String BK_TOTAL_WRITE_LATENCY = "segmentstore.bookkeeper.total_write_latency_ms";   // Including Queue. Per-container Histogram.
```

Comment:
This metric is currently reported to InfluxDB as `pravega_bookkeeper_segmentstore_bookkeeper_write_latency_ms`,
and it is bookkeeper client metric, if I am not mistaken.

Are you sure that removing `segmentstore` from here will not lead to confusion with Bookkeeper server metrics if they were to be saved to the same database?

Reply:
Agree, only the repeating bookkeeper is removed.